pkg/common/patterns_test.go

```diff
@@ -0,0 +1,33 @@
+package common
+
+import (
+	"regexp"
+	"testing"
+)
+
+const (
+	pattern = `?()/\+=\s\n`
+	regex   = `(?im)(?:user|usr)\S{0,40}?[:=\s]{1,3}[ '"=]{0,1}([^:?()/\+=\s\n]{4,40})\b`
+)
+
+func TestUsernameRegexCheck(t *testing.T) {
+	usernameRegexPat := UsernameRegexCheck(pattern)
+
+	expectedRegexPattern := regexp.MustCompile(regex)
+
+	if usernameRegexPat.compiledRegex.String() != expectedRegexPattern.String() {
+		t.Errorf("\n got %v \n want %v", usernameRegexPat.compiledRegex, expectedRegexPattern)
+	}
+
+	testString := `username = "johnsmith123" \n username='johnsmith123' username:="johnsmith123" username = johnsmith123 username=dustin123`
```

Comment:
This would be a lot more readable and also test multi-line chunks if you just made this a multi line raw string

Reply:
Also, the last username is `dustin123`. The test is probably not passing